doc/plugins/org.polarsys.capella.preferences.doc/html/06. Preferences/6.2. Model preferences.html

```diff
@@ -95,24 +99,71 @@ <h3 id="Title_Block">Title Block</h3>
 			<li>When a cell is selected, the content of the cell is displayed on the bottom of the table.</li>
 		</ul>
 		<p>
-			<img border="0" src="Images/Title_Block_DiagramPreferences.PNG"/>
+			<img height="595" width="877" border="0" src="Images/Title_Block_DiagramPreferences.PNG"/>
+		</p>
+		<ul>
+			<li>The content of each cell can be editable by double clicking on a cell.</li>
```

Comment:
The content of each cell can be ~~editable~~ edited by double clicking on ~~a cell~~ it.